deploy/helm-charts/README.md

```diff
@@ -1,26 +1,39 @@
-# Helm chart for deploying terrascan in server mode
+# Helm chart for deploying terrascan
 
-This chart deploys terrascan as a server within your kubernetes cluster. By default it runs just terrascan by itself, but, 
+This chart deploys terrascan as a server within your kubernetes cluster. By default it runs just terrascan by itself, but,
 user creates namespace and secrets.
 
 In server mode, terrascan will act both as an API server for
 performing remote scans of IAC, as well as a validating admission
 webhook for a Kubernetes cluster. Further details can be found in
 the [main documentation](https://docs.accurics.com/projects/accurics-terrascan/en/latest/).
+There are two helm charts:
+
+1. In the `server/` directory : to deploy terrascan in server mode.
+2. In the `webhook/` directory : to setup a validating webhook that uses the deployed terrascan server from step 1, as its backend.
 
 ## Usage
 ### Set up TLS certificates
 A requirement to run an admission controller is that communication
 happens over TLS. This helm chart expects to find the certificate
-at `data/server.crt` and key at `data/server.key`.
+at `server/data/server.crt` and key at `server/data/server.key`.
+If you opt to deploy the webhook as well, please copy `server/data/server.crt` at `webhook/data/server.crt`
+
+### Set up SSH config for remote repo scan
+If you're opting to utilise the remote repo scan feature,
+terrascan will require ssh capabilities to do that.
```

Comment:
we can scan open repo as well so for remote repo scan ssh is not always mandatory. If someone wishes to scan private repo then only ssh keys will be used.

Reply:
yes, I missed mentioning that part